03-03-2024 01:06:24 [info]: Novo usuário registrado com sucesso: 65e3f740885a66026484fea1
03-03-2024 01:06:37 [info]: Novo cartão de crédito registrado com sucesso: 65e3f74c885a66026484fea5
03-03-2024 01:06:37 [info]: Novo usuário registrado com sucesso: 65e3f740885a66026484fea1
03-03-2024 01:06:58 [notice]: Cartão de crédito atualizado com sucesso: 65e3f74c885a66026484fea5
03-03-2024 01:07:05 [warning]: Cartão de crédito excluído com sucesso: 65e3f74c885a66026484fea5
03-03-2024 01:07:05 [notice]: Usuário atualizado com sucesso 65e3f740885a66026484fea1
03-03-2024 01:07:18 [error]: log_erroExcluirCartaoCredito
03-03-2024 01:09:06 [info]: Novo cartão de crédito registrado com sucesso: 65e3f7e21b22328186964b92
03-03-2024 01:09:06 [info]: Novo usuário registrado com sucesso: 65e3f740885a66026484fea1
03-03-2024 01:09:13 [warning]: Cartão de crédito excluído com sucesso: 65e3f7e21b22328186964b92
03-03-2024 01:09:13 [notice]: Usuário atualizado com sucesso 65e3f740885a66026484fea1
03-03-2024 01:09:18 [error]: Erro interno do servidor: CastError: Cast to ObjectId failed for value "65e3f7e21b22328186964b92a" (type string) at path "_id" for model "CartaoCredito"
03-03-2024 01:11:04 [info]: Novo cartão de crédito registrado com sucesso: 65e3f858ed858555a768c448
03-03-2024 01:11:04 [info]: Novo usuário registrado com sucesso: 65e3f740885a66026484fea1
03-03-2024 01:11:10 [notice]: Cartão de crédito atualizado com sucesso: 65e3f858ed858555a768c448
03-03-2024 01:11:15 [warning]: Cartão de crédito excluído com sucesso: 65e3f858ed858555a768c448
03-03-2024 01:11:15 [notice]: Usuário atualizado com sucesso 65e3f740885a66026484fea1
03-03-2024 01:11:20 [error]: Erro interno do servidor: CastError: Cast to ObjectId failed for value "65e3f858ed858555a768c448a" (type string) at path "_id" for model "CartaoCredito"
03-03-2024 01:11:51 [info]: Novo cartão de crédito registrado com sucesso: 65e3f887ed858555a768c456
03-03-2024 01:11:51 [info]: Novo usuário registrado com sucesso: 65e3f740885a66026484fea1
03-03-2024 01:15:33 [info]: Novo cartão de crédito registrado com sucesso: 65e3f965e22d5863a4b6333f
03-03-2024 01:15:33 [notice]: Cartão de crédito atualizado com sucesso: 65e3f740885a66026484fea1
03-03-2024 01:16:36 [info]: Novo cartão de crédito registrado com sucesso: 65e3f9a467e05fbe81b42864
03-03-2024 01:16:36 [notice]: Usuário atualizado com sucesso 65e3f740885a66026484fea1
03-03-2024 01:16:51 [warning]: Cartão de crédito excluído com sucesso: 65e3f9a467e05fbe81b42864
03-03-2024 01:16:51 [notice]: Usuário atualizado com sucesso 65e3f740885a66026484fea1
03-03-2024 18:27:12 [notice]: Usuário atualizado com sucesso 65e35e482e3d2bb2fa2447f2
03-03-2024 18:27:12 [notice]: Acesso realizado com sucesso: 65e35e482e3d2bb2fa2447f2
03-03-2024 18:27:34 [info]: Nova conta cadastrada com sucesso: 65e4eb46ec8e4cd291ee7288
03-03-2024 18:27:34 [notice]: Usuário atualizado com sucesso 65e35e482e3d2bb2fa2447f2
03-03-2024 18:28:05 [notice]: Conta atualizada com sucesso: 65e4eb46ec8e4cd291ee7288
03-03-2024 18:28:19 [error]: Erro interno do servidor: TypeError: Cannot read properties of null (reading '_id')
03-03-2024 18:28:34 [notice]: Cartão de crédito atualizado com sucesso: 65e3f965e22d5863a4b6333f
03-03-2024 18:28:38 [error]: Erro interno do servidor: CastError: Cast to ObjectId failed for value "65e3f965e22d5863a4b6333fa" (type string) at path "_id" for model "CartaoCredito"
03-03-2024 18:35:34 [error]: Erro interno do servidor.
03-03-2024 18:36:03 [notice]: Cartão de crédito atualizado com sucesso: 65e3f965e22d5863a4b6333f
03-03-2024 18:36:07 [error]: Erro interno do servidor.
03-03-2024 18:39:42 [error]: Erro interno do servidor. CastError: Cast to ObjectId failed for value "65e3f965e22d5863a4b6333fa" (type string) at path "_id" for model "CartaoCredito"/nCastError: Cast to ObjectId failed for value "65e3f965e22d5863a4b6333fa" (type string) at path "_id" for model "CartaoCredito"
    at SchemaObjectId.cast (C:\GitHub\laurus\server\node_modules\mongoose\lib\schema\objectId.js:250:11)
    at SchemaObjectId.SchemaType.applySetters (C:\GitHub\laurus\server\node_modules\mongoose\lib\schemaType.js:1219:12)
    at SchemaObjectId.SchemaType.castForQuery (C:\GitHub\laurus\server\node_modules\mongoose\lib\schemaType.js:1633:15)
    at cast (C:\GitHub\laurus\server\node_modules\mongoose\lib\cast.js:375:32)
    at model.Query.Query.cast (C:\GitHub\laurus\server\node_modules\mongoose\lib\query.js:4768:12)
    at model.Query.Query._castConditions (C:\GitHub\laurus\server\node_modules\mongoose\lib\query.js:2200:10)
    at model.Query._findOneAndUpdate (C:\GitHub\laurus\server\node_modules\mongoose\lib\query.js:3242:8)
    at model.Query.exec (C:\GitHub\laurus\server\node_modules\mongoose\lib\query.js:4290:80)
    at processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async atualizarCartaoCredito (C:\GitHub\laurus\server\src\controllers\cartaoCreditoController.ts:99:45)
03-03-2024 18:46:28 [info]: Novo cartão de crédito registrado com sucesso: 65e4efb41f1528dddf824e99
03-03-2024 18:46:28 [notice]: Usuário atualizado com sucesso 65e3f740885a66026484fea1
03-03-2024 18:46:45 [error]: Erro interno do servidor. CastError: Cast to ObjectId failed for value "65e4efb41f65e4efb41f1528dddf824e991528dddf82ae99" (type string) at path "_id" for model "CartaoCredito"
    at SchemaObjectId.cast (C:\GitHub\laurus\server\node_modules\mongoose\lib\schema\objectId.js:250:11)
    at SchemaObjectId.SchemaType.applySetters (C:\GitHub\laurus\server\node_modules\mongoose\lib\schemaType.js:1219:12)
    at SchemaObjectId.SchemaType.castForQuery (C:\GitHub\laurus\server\node_modules\mongoose\lib\schemaType.js:1633:15)
    at cast (C:\GitHub\laurus\server\node_modules\mongoose\lib\cast.js:375:32)
    at model.Query.Query.cast (C:\GitHub\laurus\server\node_modules\mongoose\lib\query.js:4768:12)
    at model.Query.Query._castConditions (C:\GitHub\laurus\server\node_modules\mongoose\lib\query.js:2200:10)
    at model.Query._findOne (C:\GitHub\laurus\server\node_modules\mongoose\lib\query.js:2484:8)
    at model.Query.exec (C:\GitHub\laurus\server\node_modules\mongoose\lib\query.js:4290:80)
    at processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async obterCartaoCreditoPorId (C:\GitHub\laurus\server\src\controllers\cartaoCreditoController.ts:76:35)
03-03-2024 18:47:11 [notice]: Cartão de crédito atualizado com sucesso: 65e4efb41f1528dddf824e99
03-03-2024 18:47:14 [error]: Erro interno do servidor. CastError: Cast to ObjectId failed for value "65e4efb41f1528dddf824e99a" (type string) at path "_id" for model "CartaoCredito"
    at SchemaObjectId.cast (C:\GitHub\laurus\server\node_modules\mongoose\lib\schema\objectId.js:250:11)
    at SchemaObjectId.SchemaType.applySetters (C:\GitHub\laurus\server\node_modules\mongoose\lib\schemaType.js:1219:12)
    at SchemaObjectId.SchemaType.castForQuery (C:\GitHub\laurus\server\node_modules\mongoose\lib\schemaType.js:1633:15)
    at cast (C:\GitHub\laurus\server\node_modules\mongoose\lib\cast.js:375:32)
    at model.Query.Query.cast (C:\GitHub\laurus\server\node_modules\mongoose\lib\query.js:4768:12)
    at model.Query.Query._castConditions (C:\GitHub\laurus\server\node_modules\mongoose\lib\query.js:2200:10)
    at model.Query._findOneAndUpdate (C:\GitHub\laurus\server\node_modules\mongoose\lib\query.js:3242:8)
    at model.Query.exec (C:\GitHub\laurus\server\node_modules\mongoose\lib\query.js:4290:80)
    at processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async atualizarCartaoCredito (C:\GitHub\laurus\server\src\controllers\cartaoCreditoController.ts:99:45)
03-03-2024 18:47:32 [error]: Erro interno do servidor. CastError: Cast to ObjectId failed for value "65e4efb41f1528dddf824e99v" (type string) at path "_id" for model "CartaoCredito"
    at SchemaObjectId.cast (C:\GitHub\laurus\server\node_modules\mongoose\lib\schema\objectId.js:250:11)
    at SchemaObjectId.SchemaType.applySetters (C:\GitHub\laurus\server\node_modules\mongoose\lib\schemaType.js:1219:12)
    at SchemaObjectId.SchemaType.castForQuery (C:\GitHub\laurus\server\node_modules\mongoose\lib\schemaType.js:1633:15)
    at cast (C:\GitHub\laurus\server\node_modules\mongoose\lib\cast.js:375:32)
    at model.Query.Query.cast (C:\GitHub\laurus\server\node_modules\mongoose\lib\query.js:4768:12)
    at model.Query.Query._castConditions (C:\GitHub\laurus\server\node_modules\mongoose\lib\query.js:2200:10)
    at model.Query._findOne (C:\GitHub\laurus\server\node_modules\mongoose\lib\query.js:2484:8)
    at model.Query.exec (C:\GitHub\laurus\server\node_modules\mongoose\lib\query.js:4290:80)
    at processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async obterCartaoCreditoPorId (C:\GitHub\laurus\server\src\controllers\cartaoCreditoController.ts:76:35)
03-03-2024 19:55:27 [error]: Erro interno do servidor. CastError: Cast to ObjectId failed for value "65e4eb46ec8e4cd291ee7288a" (type string) at path "_id" for model "CartaoCredito"
    at SchemaObjectId.cast (C:\GitHub\laurus\server\node_modules\mongoose\lib\schema\objectId.js:250:11)
    at SchemaObjectId.SchemaType.applySetters (C:\GitHub\laurus\server\node_modules\mongoose\lib\schemaType.js:1219:12)
    at SchemaObjectId.SchemaType.castForQuery (C:\GitHub\laurus\server\node_modules\mongoose\lib\schemaType.js:1633:15)
    at cast (C:\GitHub\laurus\server\node_modules\mongoose\lib\cast.js:375:32)
    at model.Query.Query.cast (C:\GitHub\laurus\server\node_modules\mongoose\lib\query.js:4768:12)
    at model.Query.Query._castConditions (C:\GitHub\laurus\server\node_modules\mongoose\lib\query.js:2200:10)
    at model.Query._findOne (C:\GitHub\laurus\server\node_modules\mongoose\lib\query.js:2484:8)
    at model.Query.exec (C:\GitHub\laurus\server\node_modules\mongoose\lib\query.js:4290:80)
    at processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async excluirCartaoCredito (C:\GitHub\laurus\server\src\controllers\cartaoCreditoController.ts:117:35)
03-03-2024 19:55:56 [warning]: Cartão de crédito excluído com sucesso: 65e4efb41f1528dddf824e99
03-03-2024 19:55:56 [notice]: Usuário atualizado com sucesso 65e3f740885a66026484fea1
03-03-2024 20:37:13 [warning]: Tentativa de acesso com e-mail inexistente: pudaao@dota2.com
03-03-2024 20:37:23 [warning]: Tentativa de acesso com senha incorreta: pudao@dota2.com
03-03-2024 20:37:59 [warning]: Tentativa de acesso com senha incorreta: pudao@dota2.com
